tests/python/shared/fixtures/data.py

```diff
@@ -357,8 +357,17 @@ def find(jobs):
 
 @pytest.fixture(scope="session")
 def filter_tasks_with_shapes(annotations):
-    def find(tasks):
-        return list(filter(lambda t: annotations["task"][str(t["id"])]["shapes"], tasks))
+    def find(tasks, skip_tasks=None):
+        def _is_task_with_shapes(t):
+            if (
+                skip_tasks is not None
+                and t["id"] in skip_tasks
+                or not annotations["task"][str(t["id"])]["shapes"]
+            ):
+                return False
+            return True
```

Comment:
Can we use such solution?
```suggestion
    def find(tasks):
        def _is_task_with_shapes(t):
            if (
                not t.get("data")
                or not annotations["task"][str(t["id"])]["shapes"]
            ):
                return False
            return True
```

Reply:
ok, thanks